src/js/components/Carousel/stories/Custom.js

```diff
@@ -1,44 +0,0 @@
-import React from 'react';
-import { Attraction, Car, TreeOption } from 'grommet-icons';
-
-import { Grommet, Box, Carousel } from 'grommet';
-
-const customTheme = {
```

Comment:
Should we have a custom theme directory and story? Something that swaps out icons, icon color, and animation duration?

```
carousel: {
      icons: {
        current: Subtract,
        next: Next,
        previous: Previous,
        // color: { dark: undefined, light: undefined },
      },
      animation: {
        duration: 1000,
      },
      disabled: {
        icons: {
          // color: { dark: undefined, light: undefined },
        },
      },
    },
```

Reply:
Write as typescript